lib/components/ActiveTextTruncate.vue

```diff
@@ -64,61 +67,61 @@
       this.$options.resizeObserver = null
     },
     computed: {
-      wrapperElement () {
+      wrapperElement ():  Element | null {
         const selector = '.active-text-truncate__wrapper'
         return this.resizeObserverKey ? this.$el.querySelector(selector) : null
       },
-      wrapperElementWidth () {
+      wrapperElementWidth (): number {
         return get(this, 'wrapperElement.offsetWidth', 0)
       },
       textElement () {
         const selector = '.active-text-truncate__wrapper__text'
         return this.resizeObserverKey ? this.$el.querySelector(selector) : null
       },
-      textElementWidth () {
+      textElementWidth (): number {
         return get(this, 'textElement.offsetWidth', 0)
       },
-      textOffsetTransitionDelay () {
+      textOffsetTransitionDelay (): string {
         return `${this.delay}ms`
       },
-      textOffsetTransitionDuration () {
+      textOffsetTransitionDuration (): string {
         const offset = Math.abs(this.wrapperElementWidth - this.textElementWidth)
         const duration = offset / this.ppms
         return `${duration}ms`
       },
-      textInitialOffset () {
-        return 0
+      textInitialOffset (): string {
+        return "0"
```

Comment:
I'm curious why you changed this to be a string ? :)

Reply:
The `textInitialOffset` is a CSS property value so it is handle as text, and using the "+" operators between strings and numbers can be surprising (to say the least) in JS because of type inference so using only strings is safer.

Reply:
:+1: